.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: ruby
 rvm:
 - '2.2.5'
-sudo: false
+sudo: true
 cache: bundler
 env:
   global:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
+  - PARALLEL_TEST_PROCESSORS=3
```

Comment:
playing with vm infra on travis with 3 cores... maybe more?  Note, 2 on containers took ~26 minutes for vmdb suite.